### PR TITLE
Tranlate to German

### PR DESCRIPTION
**Pull request by AKingUltra**
Translation into German, one question remains: Do you want formal or informal language? This translation is in the formal style, please let me know if informal is preferred.

### CODE REVIEW

**Comment by tmashuang** [comment]
We have made added more to the `messages.json` file, please update and add.

**Comment by kumavis** [comment]
@AKingUltra we've been preferring more informal language
also if you want to use quotations (`"`) in translations, you'll need to escape them (`\"`).

File: app/_locales/de/messages.json [hunk 1]
```diff
@@ -0,0 +1,609 @@
...
+  },
+  "pleaseReviewTransaction": {
+    "message": "Bitte überprüfen Sie Ihre Transaktion."
+  },
+  "privateKey": {
+    "message": "Private Key",
+    "description": "Wählen Sie diesen Dateityp um damit einen Account zu importieren"
+  },
+  "privateKeyWarning": {
+    "message": "Warnung: Niemals jemanden den Private Key mitteilen. Jeder der im Besitzt Ihres Private Keys ist, kann jegliches Guthaben Ihres Accounts stehlen.
```
Comment (kumavis) [inline app/_locales/de/messages.json]: need to close the string `"`

File: app/_locales/de/messages.json [hunk 2]
```diff
@@ -0,0 +1,609 @@
...
+  },
+  "testFaucet": {
+    "message": "Testfaucet"
+  },
+  "to": {
+    "message": "An"
+  },
+  "toETHviaShapeShift": {
+    "message": "$1 an ETH via ShapeShift",
+    "description": "Das System wird den Einzahlungstyp im Beginn der Nachricht eintragen“
```
Comment (kumavis) [inline app/_locales/de/messages.json]: need to close the string with a normal quotation mark (`"`)

**Review by kumavis — DISMISSED** [review]

**Comment by AKingUltra** [comment]
Hi kumavis. Alrighty, I'll change it tomorrow and will fix the mistakes. Cheerio!

File: app/_locales/de/messages.json [hunk 3]
```diff
@@ -0,0 +1,609 @@
...
+    "message": "Warnung: Niemals jemanden den Private Key mitteilen. Jeder der im Besitzt Ihres Private Keys ist, kann jegliches Guthaben Ihres Accounts stehlen.  
+  },
+  "privateNetwork": {
+    "message": "Privates Netzwerk"
+  },
+  "qrCode": {
+    "message": "QR Code anzeigen"
+  },
+  "readdToken": {
+    "message": "Sie können diesen Token zukünftig wieder hinzufügen indem Sie in den Menüpunkt "Token hinzufügen" in den Einstellungen Ihres Accounts gehen."
```
Comment (kumavis) [inline app/_locales/de/messages.json]: need to escape quotes with the slash
```
\"Token hinzufügen\"
```

**Comment by AKingUltra** [comment]
Hi again, 
I just checked out the Beta in German more thoroughly and I found a few missing translations, formatting and graphic errors. I don't know if this is the right place for this but I uploaded my findings and some screenshots.
[Errors.xlsx](https://github.com/MetaMask/metamask-extension/files/1859301/Errors.xlsx)

**Pull request merged**